src/space.cpp

```diff
@@ -2215,6 +2215,14 @@ static iarf_e do_space(chunk_t *first, chunk_t *second, int &min_sp)
       return(options::sp_after_noexcept());
    }
 
+   // Issue #2138
+   if (  chunk_is_token(first, CT_FPAREN_CLOSE)
+      && chunk_is_token(second, CT_QUALIFIER))
```

Comment:
While we're at it, maybe we should also add/remove space in `foo()noxecept`, either controlled by the same option, or by a similar option?